autorest/codegen/models/enum_schema.py

```diff
@@ -31,9 +32,10 @@ def from_yaml(cls, yaml_data: Dict[str, Any]) -> "EnumValue":
         :return: A created EnumValue
         :rtype: ~autorest.models.EnumValue
         """
+        value = yaml_data["value"]
         return cls(
             name=yaml_data["language"]["python"]["name"],
-            value=yaml_data["value"],
+            value=f"\"{value}\"" if need_quotes else value,
```

Comment:
Still unhappy, as I told you `get_declaration` should take care of that

Reply:
sorry I totally missed all of your comments before I merged the testing commit, working on them rn. Sorry for the re-review!